front/lib/models/plan.ts

```diff
@@ -117,7 +117,7 @@ export class Subscription extends Model<
   declare updatedAt: CreationOptional<Date>;
 
   declare sId: string; // unique
-  declare status: string; // "active" | "ended"
+  declare status: string; // "active" | "ended" | "cancelled"
```

Comment:
Should we just type it "active" | "ended" | "cancelled" so typescript can save us ?